internal/util/functions/functions_test.go

```diff
@@ -0,0 +1,198 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package functions_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/GoogleContainerTools/kpt/internal/util/functions"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestReconcileFunctions(t *testing.T) {
```

Comment:
Shouldn't we convert this to table tests.

Reply:
Done